runtime/include/jvmti.h.m4

```diff
@@ -341,7 +341,10 @@ typedef enum jvmtiError {
 	JVMTI_ERROR_NAMES_DONT_MATCH = 69,
 	JVMTI_ERROR_UNSUPPORTED_REDEFINITION_CLASS_MODIFIERS_CHANGED = 70,
 	JVMTI_ERROR_UNSUPPORTED_REDEFINITION_METHOD_MODIFIERS_CHANGED = 71,
-	JVMTI_ERROR_UNSUPPORTED_REDEFINITION_CLASS_ATTRIBUTE_CHANGED = 72,
+dnl m4 can't understand (defined(J9VM_OPT_VALHALLA_NESTMATES))
```

Comment:
This line can be removed.  Nestmates is an 11 feature so we don't need the specific ifdef for it anymore

Reply:
Perhaps it's time to remove all appearances of `J9VM_OPT_VALHALLA_NESTMATES` (in a separate PR)?

Reply:
Agreed.  They should all become Java 11 version checks

Reply:
@DanHeidinga Do you also agree that change should be in a separate pull request or would you prefer I add those changes here?

Reply:
My assumption, and sorry if it wasn't clear, is that we'd delete the lines in this PR and the larger cleanup of `J9VM_OPT_VALHALLA_NESTMATES` can be deferred to another issue

Reply:
Done.